playbook/workshop_project_model.md

```diff
@@ -0,0 +1,73 @@
+# Club Structure
+
+People seem to be more motivated when they get to choose what they do. Ideally
+therefore, a Hack Club would be formatted like a maker space. Hack Club would be
+a space for people to come in and hack on projects together. However this does
+not work for complete beginners because they don't know where to start.
+
+One solution is only to teach them the basics through lessons or workshops for
+the first several months. That's about as exciting as math teachers saying you
+have to take years of math before you get to do anything interesting with it. We
+want the beginner to be able to build their own project as soon as possible.
+Therefore we want them to learn just enough to build their own projects and then
+as soon as possible, actually build their own project. This builds confidence
+and motivation in the beginner.
+
+At [Hack Camp](https://camp.hackedu.us/), we repeated a 2-day cycle where:
+
+1. learners spend the first 2 hours of their first day following a workshop
+2. for the remainder of the first day and all of the second day, they would
```

Comment:
This ordered list reads like a sentence. What if the numbering were taken out entirely?